modin/test/interchange/dataframe_protocol/hdk/test_protocol.py

```diff
@@ -30,6 +30,9 @@
 from .utils import get_data_of_all_types, split_df_into_chunks, export_frame
 
 
+@pytest.mark.xfail(
+    reason="conversion from 'pyarrow' ends up with the wrong datetime64 resolution"
+)
```

Comment:
there's an `exclude_dtypes` parameter in `get_data_of_all_types()`, can we simply exclude datetimes instead of disabling all these tests?